doc/source/whatsnew/v1.0.0.rst

```diff
@@ -186,6 +186,7 @@ Sparse
 
 Build Changes
 ^^^^^^^^^^^^^
+- Fixed pyqt development dependency (:issue:`26838`)
```

Comment:
can you be a little more specific here; IOW what would someone reading these notes need to know about this change

Reply:
yeah, make sense, will change later today! @jreback

Reply:
as well as that type